lib/inspect.py

```diff
@@ -384,7 +385,9 @@ def convert_sarif(app_name, repo_context, sarif_files, findings_fname):
                             }
                             findings_list.append(finding)
                             finding_id = finding_id + 1
-        try:
-            json.dump({"findings": findings_list}, out_file)
-        except Exception:
-            LOG.debug("Unable to convert the run to findings format")
+            except Exception as e:
+                LOG.debug(e)
```

Comment:
maybe it should be a warn?

Reply:
SCAN_DEBUG_MODE=debug will print debug. Any message from scan is resulting in an email or a github ticket which I want to avoid. So scan by default doesn't print anything :)